controller/area_blackbox_test.go

```diff
@@ -347,32 +311,19 @@ func newCreateChildAreaPayload(name string) *app.CreateChildAreaPayload {
 	}
 }
 
-func createSpaceAndArea(t *testing.T, db *gormapplication.GormDB) (space.Space, area.Area) {
-	var areaObj area.Area
-	var spaceObj space.Space
-	application.Transactional(db, func(app application.Application) error {
-		owner := &account.Identity{
-			Username:     "new-space-owner-identity",
-			ProviderType: account.KeycloakIDP,
-		}
-		errCreateOwner := app.Identities().Create(context.Background(), owner)
-		require.NoError(t, errCreateOwner)
-
-		spaceObj = space.Space{
-			Name:    "TestAreaREST-" + uuid.NewV4().String(),
-			OwnerID: owner.ID,
-		}
-		_, err := app.Spaces().Create(context.Background(), &spaceObj)
-		require.NoError(t, err)
-		name := "Main Area-" + uuid.NewV4().String()
-		areaObj = area.Area{
-			Name:    name,
-			SpaceID: spaceObj.ID,
-		}
-		err = app.Areas().Create(context.Background(), &areaObj)
-		require.NoError(t, err)
-		return nil
-	})
+func createSpaceAndArea(t *testing.T, db *gorm.DB) (space.Space, area.Area) {
+	fxt := tf.NewTestFixture(t, db,
+		tf.Identities(1, tf.SetIdentityUsernames("new-space-owner-identity")),
+		tf.Spaces(1, func(fxt *tf.TestFixture, idx int) error {
+			fxt.Spaces[idx].Name = "Test Space-" + uuid.NewV4().String()
+			fxt.Spaces[idx].OwnerID = fxt.Identities[idx].ID
```

Comment:
No need to set the owner. It should be done automatically as seen here: https://github.com/fabric8-services/fabric8-wit/blob/master/test/testfixture/make_functions.go#L81